internal/metrics/metrics.go

```diff
@@ -16,16 +16,28 @@
 
 package metrics
 
-import "github.com/prometheus/client_golang/prometheus"
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
 
 var registry *prometheus.Registry
+var BatchPinCounter prometheus.Counter
+
+// MetricsBatchPin is the prometheus metric for total number of batch pins submitted
+var MetricsBatchPin = "ff_batchPin_total"
```

Comment:
Pedantic, but would either go all lowercase `batchpin` or `batch_pin` just based on what I've seen metrics normally named in other projects.

Reply:
`batchpin` works for me, thanks 🙏🏻